docs/guides/caching.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Caching in Cortex"
+linkTitle: "Caching in Cortex"
+weight: 1
+slug: caching
+---
+
+# Caching in Cortex
```

Comment:
Please remove this line, otherwise you end up with double title:
```suggestion
```

Ie.
![Screen Shot 2020-03-03 at 10 32 25](https://user-images.githubusercontent.com/1701904/75762006-4bd5a380-5d3a-11ea-9f28-41e1742338d0.png)

Reply:
How do you get the docs to build locally?

Reply:
There's some initial setup to be done (ie. install Hugo). There's a basic documentation in `CONTRIBUTING.md`, but I haven't checked how updated it is.

Then:
```
# Run every time you change the doc
./tools/website/web-pre.sh

# Keep it running in another shell
make web-serve
```

Reply:
_I'm going to retry the local setup and submit a doc PR. Looks me and Cyril are the only one with building the website locally._